src/Opdex.Platform.Infrastructure/Clients/SignalR/PlatformHub.cs

```diff
@@ -16,8 +15,9 @@ public class PlatformHub : Hub<IPlatformClient>
     public PlatformHub(ITwoWayEncryptionProvider twoWayEncryptionProvider, AuthConfiguration authConfiguration, OpdexConfiguration opdexConfiguration)
     {
         _twoWayEncryptionProvider = twoWayEncryptionProvider ?? throw new ArgumentNullException(nameof(twoWayEncryptionProvider));
-        var authCallbackUri = new Uri(Path.Combine(opdexConfiguration.ApiUrl, authConfiguration.StratisSignatureAuth.CallbackPath));
-        _authCallback = $"{authCallbackUri.Authority}{authCallbackUri.AbsolutePath}";
+        var created = Uri.TryCreate($"{opdexConfiguration.ApiUrl}/{authConfiguration.StratisSignatureAuth.CallbackPath}", UriKind.Absolute, out var uri);
+        if (!created) throw new Exception("Unable to create callback URI");
+        _authCallback = $"{uri.Authority}{uri.AbsolutePath}";
```

Comment:
Believe that `Uri.TryCreate` will only work as expected if `opdexConfiguration.ApiUrl` doesn't contain a path. As it will trim any path on the first parameter. But I'm not sure this is going to be a problem.

Can you do the same for the endpoint in the auth controller? That builds the callback URL with `Path.Combine` as well